src/modeles/actionSaisie.js

```diff
@@ -22,30 +23,37 @@ class ActionSaisie extends Base {
   }
 
   description() {
-    return this.referentiel.descriptionActionSaisie(this.id);
+    return this.referentiel.descriptionActionSaisie(this.version, this.id);
   }
 
   suivante() {
-    return this.referentiel.actionSuivante(this.id);
+    return this.referentiel.actionSuivante(this.version, this.id);
   }
 
   toJSON() {
     return {
       id: this.id,
       description: this.description(),
       statut: this.homologation.statutSaisie(this.id),
```

Comment:
⚠️ Si mon modèle mental est correct par rapport à cette ligne : 
- on ne pourra pas utiliser des `id` différents pour les actions `"v2"`
- on ne pourra pas remanier vers `this.homologation.statutSaisie(this.version, this.id)` au risque de casser les statuts déjà valorisés en PROD

Reply:
Les `id` doivent effectivement représenter un des attributs de la classe `Homologation` – donc, `descriptionService`, `mesures` fonctionnent dès maintenant. Pour `homologations` ça va être un peu plus coton tout de suite… mais on devrait pouvoir changer cette ligne en –

```javascript
statut: this.inactif ? InformationHomologation.A_SAISIR : this.homologation.statutSaisie(this.id)
```

(Bien sûr, ça pourra valoit le coup d'extraire ce calcul dans une fonction et le tester à part.)

Reply:
Tout à fait d'accord avec la ternaire que tu proposes au-dessus.   
Dans mon plan, elle arriverait avec la prochaine PR qui ajoute les `'v2'`.

--

> Ce n'est pas clair pour moi de pourquoi on voudrait remanier vers ça… 🤔

En effet je pense qu'on ne veut pas... Je voulais attirer l'attention sur le risque lié à « *il faut bien veiller à ce que les `id` `'v2'` soient identiques aux `id` `'v1'`*».